pkg/services/ngalert/api/util.go

```diff
@@ -13,3 +15,59 @@ func toMacaronPath(path string) string {
 		return []byte(fmt.Sprintf(":%s", m))
 	}))
 }
+
+func timePtr(t strfmt.DateTime) *strfmt.DateTime {
+	return &t
+}
+
+func float32Ptr(f float32) *float32 {
+	return &f
+}
+
+func float64Ptr(f float64) *float64 {
+	return &f
+}
+
+func int8Ptr(i int8) *int8 {
+	return &i
+}
+
+func int16Ptr(i int16) *int16 {
+	return &i
+}
+
+func int32Ptr(i int32) *int32 {
+	return &i
+}
+
+func int64Ptr(i int64) *int64 {
+	return &i
+}
+
+func uint8Ptr(ui uint8) *uint8 {
+	return &ui
+}
+
+func uint16Ptr(ui uint16) *uint16 {
+	return &ui
+}
+
+func uint32Ptr(ui uint32) *uint32 {
+	return &ui
+}
+
+func uint64Ptr(ui uint64) *uint64 {
+	return &ui
+}
+
+func stringPtr(s string) *string {
+	return &s
+}
+
+func boolPtr(b bool) *bool {
+	return &b
+}
+
+func anyPtr(v interface{}) *interface{} {
```

Comment:
This seems unused and what's the need for `*interface{}`?

Reply:
None, I just copied and pasted it the others 😳